### PR TITLE
Don't hide edit button if there is no value

### PR DESCRIPTION
**Pull request by onEXHovia**
## Changelog

```markdown
### Added
- Don't hide edit button `sonata_type_model_list_widget` if there is no value
```

### CODE REVIEW

**Comment by OskarStark** [comment]
Please describe your problem more in detail

Thank you

**Comment by onEXHovia** [comment]
Hi @OskarStark , leave the description in the bound PR https://github.com/sonata-project/SonataAdminBundle/pull/5327

File: src/Resources/views/Form/form_admin_fields.html.twig [hunk 1]
```diff
@@ -98,12 +98,12 @@ file that was distributed with this source code.
                 {% if sonata_admin.field_description.associationadmin.hasroute('edit')
                     and sonata_admin.field_description.associationadmin.isGranted('EDIT')
                     and btn_edit
-                    and sonata_admin.value != null
                 %}
-                    <a  href="{{ sonata_admin.field_description.associationadmin.generateUrl('edit', {'id' : sonata_admin.field_description.associationadmin.normalizedIdentifier(sonata_admin.value)}) }}"
+                    <a  href="{{ sonata_admin.value == null ? '' : sonata_admin.field_description.associationadmin.generateUrl('edit', {'id' : sonata_admin.field_description.associationadmin.normalizedIdentifier(sonata_admin.value)}) }}"
```
Comment (greg0ire) [inline src/Resources/views/Form/form_admin_fields.html.twig]:
```suggestion
                    <a  href="{{ sonata_admin.value == null ? '' : sonata_admin.field_description.associationadmin.generateUrl('edit', {
                        'id' : sonata_admin.field_description.associationadmin.normalizedIdentifier(sonata_admin.value)
                    }) }}"
```

File: src/Resources/views/Form/form_admin_fields.html.twig [hunk 2]
```diff
@@ -98,12 +98,12 @@ file that was distributed with this source code.
...
                         onclick="return start_field_dialog_form_edit_{{ id }}(this);"
                         class="btn btn-warning btn-sm sonata-ba-action"
                         title="{{ btn_edit|trans({}, btn_catalogue) }}"
+                        style="{{ sonata_admin.value == null ? 'display: none;' : '' }}"
```
Comment (greg0ire) [inline src/Resources/views/Form/form_admin_fields.html.twig]: Isn't there a class in bootstrap to achieve this?

**Comment by greg0ire** [comment]
Not approving this until I understand what it is about, sorry.

**Comment by core23** [comment]
Can you provide a before / after screenshot?

**Comment by core23** [comment]
Ping @onEXHovia

**Comment by onEXHovia** [comment]
@core23 ✋, problem is after adding the value, sometimes you need to edited, but the edit button is available only after updating the object.

My case was this:

1) Image upload (SonataMedia)
2) Show editing button
3) In modal editing window, I do cropping image and save the changes

It looks something like this:
49433181-2bcfd100-f7d3-11e8-9076-4104496d2da8

Without this PR and https://github.com/sonata-project/SonataAdminBundle/pull/5327, the edit button will not be displayed after the selected value.

**Pull request merged**